test/index.js

```diff
@@ -130,37 +130,6 @@ ava.test('front pressed error is stored correctly', function (test) {
     ));
 });
 
-ava.test('dynamo DB error makes the lambda fail', function (test) {
```

Comment:
Is this some functionality that we've removed?

Reply:
This test has been broken for over a year or two, and having looked into it doesn't seem to be obviously fixable. I've checked out the commit that had a successful run last [here](https://travis-ci.org/guardian/front-pressed-lambda/builds/380705375?utm_medium=notification&utm_source=github_status) and it still fails on my local machine, alongside others.

So, I think removing it is okay, as it appears to a flaky test reliant on external issues rather than internal logic. What do you think?

Reply:
I was mainly unsure what it was testing and whether it related to the changes this PR implements.

If we're not getting any value from it, it makes sense to get rid of it.